code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1955,3 +1955,24 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 			to_chat(usr, "[src] does not have any stored infomation!")
 	else
 		to_chat(usr, "OOC Metadata is not supported by this server!")
+
+/mob/living/carbon/human/ObjBump(A)
+	if(HAS_TRAIT(src, TRAIT_FORCE_DOORS))
+		if(istype(A, /obj/machinery/door/airlock))
```

Comment:
Would this be better on `/obj/machinery/door/proc/bumpopen()`? Since it's currently only affecting doors and nothing else it seems a bit overkill to check on everything. Then again, this will only be on a couple of vampire players, and only if they have the ability enabled.